candi/cloud-providers/yandex/terraform-modules/vpc-components/nat_instance.tf

```
@@ -88,6 +88,8 @@ locals {
     EOF
 
     netplan apply
+
+    sysctl -w net.netfilter.nf_conntrack_max=1048576
```

```suggestion
    sysctl -w net.netfilter.nf_conntrack_max=1048576
    sysctl -w net.netfilter.nf_conntrack_tcp_timeout_time_wait=30
```
@zuzzas What do you think about having `nf_conntrack_tcp_timeout_time_wait` for all yandex nat instances?

---

No objections at all.

We've messed around with it with @stratnevpy. Worked perfectly well. It will affect only those applications that use TCP and do not implement any kind of keepalive. And that's completely A-OK.

---

@zuzzas @konstantin-axenov maybe we could also decrease `net.netfilter.nf_conntrack_tcp_timeout_fin_wait` and `net.netfilter.nf_conntrack_tcp_timeout_close_wait` to 30 too?